fastify.js

```diff
@@ -583,6 +584,18 @@ function buildRoutePrefix (instancePrefix, pluginPrefix) {
   return instancePrefix + pluginPrefix
 }
 
+function wrapRouting (httpHandler, { rewriteUrl, logger }) {
+  if (!rewriteUrl) {
+    return httpHandler
+  }
+  return function preRouting (req, res) {
+    const originalUrl = req.url
+    req.url = rewriteUrl(req)
+    logger.debug({ originalUrl, url: req.url }, 'rewrite url')
```

Comment:
I would add a condition to log only if `originalUrl !== req.url` otherwise this log could be noisily

What do you think?

Reply:
i don't think anyone will log this in production as it's debug level
it's useful for development though even if it was a noop i think